### PR TITLE
Add azure static website component

### PR DESCRIPTION
**Pull request by sean1588**
fixes: https://github.com/pulumi/growth/issues/12

### CODE REVIEW

**Review by stack72 — APPROVED** [review]

File: themes/default/content/registry/packages/azure-static-website/_index.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+title: Azure Static Website
+meta_desc: Provides an overview of the Azure Static Website component.
+layout: overview
+---
+
+This component makes it easy to deploy a static website to Azure using any of the supported Pulumi programming languages.
+
+## Usage
+
+```typescript
```
Comment (cnunciato) [inline themes/default/content/registry/packages/azure-static-website/_index.md:11]: Would you be up for placing these within a language chooser?

File: themes/default/content/registry/packages/azure-static-website/_index.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+- originURL - the Storage URL for the site
+- cdnURL - the CDN URL for the site
+- customDomainURL - the custom domain URL where the static website can be accessed
+- resourceGroupName - the name of the resource group that was provisioned to contain the needed static website resources
+
+
+## Notes:
+
+- If you would like to serve your site from a custom domain, you need to configure a DNS zone in Azure and set the name of the zone using the `dnsZoneName` input property
```
Comment (cnunciato) [inline themes/default/content/registry/packages/azure-static-website/_index.md:59]:
```suggestion
- If you would like to serve your site from a custom domain, you need to configure a DNS zone in Azure and set the name of the zone using the `dnsZoneName` input property.
```

File: themes/default/content/registry/packages/azure-static-website/installation-configuration.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+## Installation
+
+The Pulumi Azure Static Website component is available as a package in all Pulumi languages:
+
+* JavaScript/TypeScript: [`@pulumi/azure-static-website`](https://www.npmjs.com/package/@pulumi/azure-static-website)
+* Python: [`pulumi_azure_static_website`](https://pypi.org/project/pulumi-azure-static-website/)
+* Go: [`github.com/pulumi/pulumi-azure-static-website/sdk`](https://pkg.go.dev/github.com/pulumi/pulumi-azure-static-website/sdk)
+* .NET: [`Pulumi.AzureStaticWebsite`](https://www.nuget.org/packages/Pulumi.AzureStaticWebsite/)
+
+If you are new to Pulumi and do not yet have our Pulumi installed, see our [getting started](https://www.pulumi.com/docs/get-started/) guide to get up and running.
```
Comment (cnunciato) [inline themes/default/content/registry/packages/azure-static-website/installation-configuration.md:16]: This extra "our" has been copied around a bit -- I've been meaning to get in there and clear it out, but haven't had time. 😅  

```suggestion
If you are new to Pulumi and do not yet have Pulumi installed, see our [getting started](https://www.pulumi.com/docs/get-started/) guide to get up and running.
```

**Review by cnunciato — APPROVED** [review]
Couple little things, otherwise 🚀! Thanks for doing this!

**Comment by sean1588** [comment]
thank YOU for making the component! :tada: I am realizing now, I think we need to add a docs folder with the index.md and installation-configuration.md files to the azure-static-website repo before merging this. I think there is a job that runs that tries to keep those files in sync with these that are in the registry. I'll get those added.

**Pull request merged**